static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -73,6 +75,26 @@ class TeamMembers extends AsyncView<Props, State> {
     200
   );
 
+  debounceFetchTeamDetailsRequest = debounce(() =>
+    this.setState(() => this.fetchTeamDetailsRequest())
+  );
+
+  fetchTeamDetailsRequest = async () => {
```

Comment:
You shouldn't need to fetch this yourself because `Team` is a commonly used model on the frontend, so there should be a cache on the frontend that you can use. Look for the file `teamStore.tsx`.